app/javascript/components/components/Header.test.js

```diff
@@ -14,9 +14,9 @@ import Header from './Header'
 Enzyme.configure({ adapter: new Adapter() })
 
 describe("When Header renders", () => {
-  it("displays a heading", () => {
+  it("displays four NavItems", () => {
     const header = shallow(<Header />)
-    const headerHeading = header.find("h1")
-    expect(headerHeading.text()).toEqual("This should fail")
+    const headerHeading = header.find("NavItem")
+    expect(headerHeading.length).toEqual(4)
```

Comment:
Will 4 NavItems always render on the page?  Does this test pass?

Reply:
The test does pass but I could see where it wouldn't if I was logged in. I changed it to just look for the Nav and it passes.